Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I think this file can be removed from source control and added to gitignore.

Reply:
Added Package.resolved to gitignore thanks